transforms/language/pdf2parquet/python/src/pdf2parquet_local_python.py

```diff
@@ -36,9 +36,9 @@
     "runtime_job_id": "job_id",
     "runtime_code_location": ParamsUtils.convert_to_ast(code_location),
     # pdf2parquet params
-    # "pdf2parquet_do_table_structure": False,
-    # "pdf2parquet_contents_type": "application/json",
-    # "pdf2parquet_contents_type": "text/markdown",
+    #"pdf2parquet_do_table_structure": False,
+    #"pdf2parquet_contents_type": "application/json",
+    #"pdf2parquet_contents_type": "text/markdown",
 }
```

Comment:
DO we need this commented-out stuff?

Reply:
it is actually there to simplify running the transform with some common options, which are also tested individually in the tests